server/build/babel/find-config.js

```diff
@@ -1,16 +1,9 @@
 import { join } from 'path'
-import buildConfigChain from 'babel-core/lib/transformation/file/options/build-config-chain'
+import {loadPartialConfig} from '@babel/core'
 
 export default function findBabelConfig (dir) {
   // We need to provide a location of a filename inside the `dir`.
   // For the name of the file, we could be provide anything.
   const filename = join(dir, 'filename.js')
```

Comment:
This is a little concerning. We've got path-based config processing now from https://github.com/babel/babel/pull/7091, so by hard-coding a filename you're essentially making that feature unusable.

If this is done as a performance optimization, I'd caution against worrying too much there. Babel's config processing has a lot more caching now and should be very fast, even when this method is called for every single file.

Reply:
> If this is done as a performance optimization, I'd caution against worrying too much there. Babel's config processing has a lot more caching now and should be very fast, even when loadPartialConfig is called for every single file.

Basically it's not to pre-optimize, but simply to output which path babel is using when starting the Next process, to be transparent to the user.